Source/ImmutableMarketplace/Public/OnRampWebBrowserWidget.h

```diff
@@ -68,6 +67,15 @@ class IMMUTABLEMARKETPLACE_API UOnRampWidget : public UWidget
 	UPROPERTY(EditDefaultsOnly, BlueprintReadOnly)
 	FString InitialURL = TEXT("about:blank");
 
+	UPROPERTY(EditDefaultsOnly, BlueprintReadOnly)
+	EImmutableEnvironment Environment = EImmutableEnvironment::Sandbox;
+
+	UPROPERTY(EditDefaultsOnly, BlueprintReadOnly)
+	FImmutableOnRampQueryParams QueryParams;
+
+	UPROPERTY(EditDefaultsOnly, BlueprintReadOnly)
+	TMap<FString, FString> ExtraQueryParams;
```

Comment:
I’m not sure if this is the right place or if I’ve missed it, but there should be documents explaining how to use this as well as links to https://docs.transak.com/docs/query-parameters.